spec/features/users/user_updates_information_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails-helper"
+
+# As an authenticated user
+# I want to update my information
+# So that I can keep my profile up to date
+
+
+  feature "user updates user information", %{
+    As a signed in user, I want to be able to
+    update my user information
+  } do
+
+  scenario "update user information" do
+    user1 = FactoryGirl.create(:user)
+    user2 = FactoryGirl.create(:user)
+
+    visit root
+    click_button "Sign in"
+    fill_in "Email", with: user1.email
+    fill_in "Password", with: user1.password
+
+    click_button "My account"
+    fill_in "Email", with: user2.email
+    fill_in "Password", with: user2.email
+
+    click_button "Save changes"
+
+    expect(page).to have_content(user2.email)  #is there a way to show 'email field contains? user2.email'
+    expect(page).to have_content(user2.password) #same for password, assumed after  redirect back to account info?
+  end
+
+end
```

Comment:
`end` at 32, 0 is not aligned with `feature "user updates user information", %{` at 8, 2 or `} do` at 11, 2